openedx/core/djangoapps/xmodule_django/models.py

```diff
@@ -68,7 +68,7 @@ def _strip_value(value, lookup='exact'):
     return stripped_value
 
 
-class OpaqueKeyField(models.CharField):
+class OpaqueKeyField(CreatorMixin, models.CharField):
```

Comment:
Nit: It'd be nice to clarify how this field works now by providing a few simple examples in the docstring. Something like:
```
>>> thing = ModelWithOpaqueKeyField(course_key='course-v1:OrgX+SomeCourse+2017', ...)
>>> thing.course_key
<output>
>>> thing.save()
>>> thing.course_key
<output>
>>> other_thing = ModelWithOpaqueKey()
>>> other_thing.course_key
<output>
>>> other_thing.save()
<output>
```
etc......